db/schema.rb

```diff
@@ -173,7 +167,7 @@
     t.text     "comments"
     t.string   "error_code"
     t.string   "id_code"
-    t.boolean  "is_template", default: false
+    t.boolean  "is_template",                         default: false
```

Comment:
@marshall-alex unknown extra whitespace here.

Reply:
Correct me if I'm wrong, @marshall-alex but I doubt you edited this file. It is typically system generated. I would like to know what causes this behavior. Did you perhaps open it in your editor, which did some auto formatting ?

Reply:
I did not purposely make any changes to that file I'd guess my editor is probably the culprit